2022/wordle/german/sentence_translations.json

```diff
@@ -128,7 +128,7 @@
  {
   "input": "We started with crane, we got grey, yellow, green, grey, grey.",
   "model": "nmt",
-  "translatedText": "Wir fingen mit dem Kranich an, wir bekamen Grau, Gelb, Grün, Grau, Grau.",
+  "translatedText": "Wir fingen mit crane an, wir bekamen Grau, Gelb, Grün, Grau, Grau.",
```

Comment:
the words "grey, yellow, green, grey, grey" should not have been translated, they're part of the game interaction.